src/System.Net.Http/tests/FunctionalTests/TestHelper.cs

```diff
@@ -166,8 +166,7 @@ public static bool NativeHandlerSupportsSslConfiguration()
 
             // For other Unix-based systems it's true if (and only if) the currect openssl backend
             // is used with libcurl.
-            bool hasAnyOpenSsl =
-                Interop.Http.GetSslVersionDescription()?.StartsWith(Interop.Http.OpenSslDescriptionPrefix, StringComparison.OrdinalIgnoreCase) ?? false;
+            bool hasAnyOpenSsl = false;
 
             if (!hasAnyOpenSsl)
```

Comment:
Since `hasAnyOpenSsl` is now being set to `false` always, it means this 'if' block is always return. That means it will return `false` always right here. And the subject code after the 'if' block is dead code and should be removed.

Reply:
Fixed.